lib/fizzy/types.hpp

```diff
@@ -362,7 +362,7 @@ struct Code
 
     // The instructions bytecode without immediate values.
     // https://webassembly.github.io/spec/core/binary/instructions.html
-    std::vector<Instr> instructions;
```

Comment:
Shouldn't we have a `static_assert(sizeof(Instr) == sizeof(uint8_t));` in this file?

Reply:
This is not required because we don't `reinterpret_cast<Instr*>(bytes_ptr)` or the other way.

Reply:
Yeah though if it is not too much hassle having that extra assert could be helpful down the line, in case we mess it up somewhere.

Reply:
The `enum class Instr : uint8_t` is defined in this file.

Reply:
Oh right, lets hope the `uint8_t` is not removed :)